.github/workflows/release.yml

```diff
@@ -86,8 +89,14 @@ jobs:
         include:
           - os: "macos-10.15"
             triplet: "x64-osx-dynamic"
+            vcpkg_cache: "/Users/runner/.cache/vcpkg/archives"
```

Comment:
I added the asset paths with hardcoded links because they didn't work properly when using the `$VCPKG_INSTALLATION_ROOT` form, but I don't see a big downside to having them hardcoded regardless.